packages/ripple/index.tsx

```diff
@@ -326,7 +342,7 @@ export function withRipple <
       />
     );
   }
-  };
+  } as any;
```

Comment:
What is this for? We shouldn't need to cast the Ripple as any. I changed this and saw that components that use react ripple are throwing an error. Is that the error you are seeing as well? 

This is what I am seeing: https://github.com/mui-org/material-ui/issues/14487

Reply:
Yes, it is. I cast as any return value of this function, but specify a correct return value for withRipple function instead. In this case, it does not visit extra files to determine the file type

Reply:
Hmm can you add a comment just saying it is a symlink issue. I investigated into this issue, and it just seems to be a short fall of typescript.